doc/src/manual/constructors.md

```diff
@@ -119,23 +119,27 @@ existence by a call to one of the inner constructor methods provided with the ty
 some degree of enforcement of a type's invariants.
 
 If any inner constructor method is defined, no default constructor method is provided: it is presumed
-that you have supplied yourself with all the inner constructors you need. The default constructor
-is equivalent to writing your own inner constructor method that takes all of the object's fields
-as parameters (constrained to be of the correct type, if the corresponding field has a type),
-and passes them to `new`, returning the resulting object:
+that you have supplied yourself with all the inner constructors you need.
+
+The default constructor is equivalent to writing one or two of your own constructor methods that take
+all of the object's fields as arguments.  One is an inner constructor method without type constraints
+on its arguments that passes the them to `new`, returning the resulting object.  The other
```

Comment:
I find this a little confusing because the call to `new` occurs in the more-specific constructor method (e.g., `T2(::Int64)` below). Your version seems to imply it's in the one with no type constraints.

By this reasoning, the original wording is better, no?

Reply:
@timholy I believe both inner constructors call `new`.  Is that not correct?